### ISSUE TITLE
Address accessibility as part of the basic stylesheets

### ISSUE THREAD

**Issue by devraj** [anomaly/jekyll-foundation#10]
Post a review of an @anomaly project by @rachelandrew Anomaly web applications haven't taken accessibility as seriously as they should.

The foundation CSS should address this and make accessibility a core feature:

- Font sizes should be relative for screen readers
- Line heights should be [unit less](http://allthingssmitty.com/2017/01/30/nope-nope-nope-line-height-is-unitless/)
- Appropriate use of markup 
- Address Sass nesting issues that end up creating very specific rules and stylesheet bloat
- Buttons should have [names](https://dequeuniversity.com/rules/axe/2.2/button-name)

Other visual elements:
- [Check contrast](https://contrastchecker.com/)
